docs/en/reference/batch-processing.rst

```diff
@@ -23,7 +23,7 @@ especially what the strategies presented here provide help with.
 .. code-block:: php
```

Comment:
We should probably update this paragraph to let the user know that some middlewares can be memory-consuming, that they should consider keeping or removing each middleware, and that if they don't want to think too much they can just remove them all.